### PR TITLE
Setup stripe fees

### PR DESCRIPTION
**Pull request by moraleida**

### CODE REVIEW

File: src/Tribe/PUE/Checker.php [hunk 1]
```diff
@@ -171,6 +180,31 @@ public function __construct( $pue_update_url, $slug = '', $options = [], $plugin
 			$this->set_plugin_file( $plugin_file );
 			$this->set_options( $options );
 			$this->hooks();
+			$this->set_key_status_transient_name();
+		}
+
+		/**
+		 * Creates a hash for the transient name that holds the current key status
```
Comment (juanfra) [inline src/Tribe/PUE/Checker.php:187]:
```suggestion
		 * Creates a hash for the transient name that holds the current key status.
```

File: src/Tribe/PUE/Checker.php [hunk 2]
```diff
@@ -171,6 +180,31 @@ public function __construct( $pue_update_url, $slug = '', $options = [], $plugin
...
+		 *
+		 * @since TBD
+		 */
+		public function set_key_status_transient_name() {
+			$this->pue_key_status_transient_name = md5( $this->get_slug() . $this->get_site_domain() );
+		}
+
+		/**
+		 * Sets the key status transient based on the key validation check results
```
Comment (juanfra) [inline src/Tribe/PUE/Checker.php:196]:
```suggestion
		 * Sets the key status transient based on the key validation check results.
```

File: src/Tribe/PUE/Checker.php [hunk 3]
```diff
@@ -171,6 +180,31 @@ public function __construct( $pue_update_url, $slug = '', $options = [], $plugin
...
+		 *
+		 * @since TBD
+		 *
+		 * @param int $valid 0 for invalid, 1 or 2 for valid
```
Comment (juanfra) [inline src/Tribe/PUE/Checker.php:200]:
```suggestion
		 * @param int $valid 0 for invalid, 1 or 2 for valid.
```

**Review by juanfra — COMMENTED** [review]
We'll need to add a changelog entry for this one, as we're modifying things on the Checker class, and we'll need to keep a record of what we're doing.

File: src/Tribe/PUE/Checker.php [hunk 4]
```diff
@@ -171,6 +180,31 @@ public function __construct( $pue_update_url, $slug = '', $options = [], $plugin
...
+		 */
+		public function set_key_status_transient( $valid ) {
+			if ( $valid ) {
+				set_transient( $this->pue_key_status_transient_name, 'valid', $this->check_period * HOUR_IN_SECONDS );
+			} else {
+				set_transient( $this->pue_key_status_transient_name, 'invalid', $this->check_period * HOUR_IN_SECONDS );
+			}
```
Comment (rafsuntaskin) [inline src/Tribe/PUE/Checker.php:207]:
```suggestion
			$status = tribe_is_truthy( $valid ) ? 'valid' : 'invalid';
			set_transient( $this->pue_key_status_transient_name, $status, $this->check_period * HOUR_IN_SECONDS );
		
```

**Comment by moraleida** [comment]
Closing in favor of #1707

**Pull request closed without merging**